homeassistant/components/conversation/default_agent.py

```diff
@@ -657,6 +664,17 @@ def _rebuild_trigger_intents(self) -> None:
         }
 
         self._trigger_intents = Intents.from_dict(intents_dict)
+
+        # Assume slot list references are wildcards
```

Comment:
Will this assumption always hold?

Reply:
Not if we choose to expose the `{name}` and `{area}` lists like we do with the builtin sentences. Should we require some kind of pattern for wildcards?